rules/cross-platform/persistence_credential_access_modify_auth_module_or_config.toml

```diff
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2020/12/21"
+maturity = "production"
+updated_date = "2020/12/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may modify the standard authentication module for persistence via patching the normal authorization process
+or modifying the login config to allow unauthorized access or elevate privileges.
+"""
+false_positives = ["Trused System Modules Updates or SysAdmin allowed PAM Deamon configuration changes."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Modification of Standard Authentication Module or Configuration"
+references = [
+    "https://github.com/zephrax/linux-pam-backdoor",
+    "https://github.com/eurialo/pambd",
+    "http://0x90909090.blogspot.com/2016/06/creating-backdoor-in-pam-in-5-line-of.html",
+    "https://www.trendmicro.com/en_us/research/19/i/skidmap-linux-malware-uses-rootkit-capabilities-to-hide-cryptocurrency-mining-payload.html",
+]
+risk_score = 47
+rule_id = "93f47b6f-5728-4004-ba00-625083b3dcb0"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Linux", "Threat Detection", "Credential Access", "Persistence"]
+type = "query"
+
+query = '''
+event.category:file and event.type:change and
+process.name:* and 
```

Comment:
is noise significantly impacted by including `process.name:*`? just wondering if its necessary or unneeded in order to make the search less expensive.

Reply:
necessary, there are some unrelated FPs with process.name null.